easybuild/easyconfigs/n/nanodbc/nanodbc-2.14.0-GCCcore-11.2.0.eb

```
@@ -0,0 +1,30 @@
+# ll4il Lorentz Institute
+easyblock = 'CMakeMake'
+
+name = 'nanodbc'
+version = '2.14.0'
+
+homepage = 'https://github.com/nanodbc/nanodbc'
+description = "A small C++ wrapper for the native C ODBC API."
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
```

if you're depending on a GCC-level easyconfig, this easyconfig should also be (at least) at GCC level
note that the filename will need to be updated to match
```suggestion
toolchain = {'name': 'GCC', 'version': '11.2.0'}
```